frontend/src/entry/overview/FormEditPage.tsx

```diff
@@ -137,28 +148,50 @@ export default function FormEditPage({ entry, currentUserRoles, readOnly = false
             </Stack>
           </div>
           {!isEdit && (
-            <Stack direction='row' spacing={1} justifyContent='flex-end' sx={{ mb: { xs: 2 } }}>
-              <Button variant='outlined' onClick={() => setExportDialogOpen(true)}>
-                Export as PDF
-              </Button>
-              <Button variant='outlined' onClick={() => setHistoryDialogOpen(true)}>
-                View History
+            <Fragment>
+              <Button data-test='openEntryOverviewActions' variant='contained' onClick={handleActionButtonClick}>
+                Actions
               </Button>
-              {!readOnly && (
-                <Tooltip title={requiredRolesText}>
-                  <span>
-                    <Button
-                      variant='outlined'
-                      disabled={!canEdit}
-                      onClick={() => setIsEdit(!isEdit)}
-                      data-test='editEntryCardButton'
-                    >
-                      {`Edit ${EntryCardKindLabel[entry.kind]}`}
-                    </Button>
-                  </span>
-                </Tooltip>
-              )}
-            </Stack>
+              <Menu anchorEl={anchorEl} open={open} onClose={handleActionButtonClose}>
+                <MenuItem>
+                  <ListItemText
+                    onClick={() => {
+                      handleActionButtonClose()
+                      setExportDialogOpen(true)
+                    }}
+                  >
+                    Export as PDF
+                  </ListItemText>
+                </MenuItem>
+                <MenuItem>
+                  <ListItemText
+                    onClick={() => {
+                      handleActionButtonClose()
+                      setHistoryDialogOpen(true)
+                    }}
+                  >
+                    View History
+                  </ListItemText>
+                </MenuItem>
+                {!readOnly && (
+                  <Tooltip title={requiredRolesText}>
+                    <span>
```

Comment:
I don't think we need to wrap the `<MenuItem />` in a `<span />`

Reply:
Removed